README.md

```diff
@@ -49,7 +49,7 @@ If using default values, once started the app should be running at [http://local
 	```
 	OPENAI_API_KEY=sk-...
 	CHROMA_DB_PATH=../chromadb
-	CHROMA_DB_COLLECTION=../chromadb
+	CHROMA_DB_COLLECTION=chromadb
```

Comment:
Is this the correct collection name, or something else?

Reply:
Oh right, collection name can be any string starting with a lowercase letter. By default it is set to `a_dot_b_collection`.